.github/workflows/build-docs.yml

```diff
@@ -43,12 +18,25 @@ jobs:
         with:
           node-version: 10.x
 
-      # Build static Storybook and docsite
-      - run: npm install
-      - run: npm run build
+      - name: Install
+        working-directory: docs
+        run: npm install
 
-      # Deploy docsite to branch
-      - name: Push docsite build directory to branch
+      - name: Build docsite
+        working-directory: docs
+        env:
+          NODE_ENV: production
+        run: |
+          npm run build
+          touch out/.nojekyll
```

Comment:
Turns out this does need to happen after the build step because Next clears out the target directory before building

Reply:
@hakimelek FYI this step was missing, that's what broke the static assets from your issue: `touch out/.nojekyll`